examples/viewer.html

```diff
@@ -84,35 +84,35 @@
       right: 0;
       bottom: 0;
       overflow-x: hidden;
-      visibility: hidden;
+      transform: translateX(-100%);
```

Comment:
Let's do both: `visibility` and `transform`. We still don't want any chance of this iframe being painted even if some platforms have bugs.